src/unit_tests/wrappers/wazuh/wazuh_db/wdb_global_wrappers.h

```diff
@@ -53,6 +53,8 @@ cJSON* __wrap_wdb_global_select_agent_name(wdb_t *wdb, int id);
 
 cJSON* __wrap_wdb_global_select_agent_group(wdb_t *wdb, int id);
 
+cJSON* __wrap_wdb_global_get_group_agents(wdb_t *wdb, wdbc_result* status, char* group_name, int last_agent_id);
```

Comment:
Same as before about type-pointer-variable_name

Reply:
It's hard to be consistent without modifying the whole file.
I'll modify it only here.